data/json/items/generic.json

```diff
@@ -3143,5 +3143,19 @@
     "weight": 4500,
     "volume": 12,
     "bashing": 8
+  },
+  {
+    "type": "GENERIC",
+    "id": "amplifier_head",
+    "symbol": ";",
+    "color": "blue",
+    "name": "amplifier head",
+    "name_plural": "amplifier heads",
```

Comment:
You can remove this as game automatically adds trailing `s` to the `name` field if no `name_plural` field is set.

Reply:
Thanks for that, I'll take those entries out in the next set of commits!